lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -453,7 +453,8 @@ SwiftLanguageRuntimeImpl::GetMemoryReader() {
   if (!m_memory_reader_sp) {
     m_memory_reader_sp.reset(new LLDBMemoryReader(
         m_process, [&](swift::remote::RemoteAbsolutePointer pointer) {
-          auto *reflection_context = GetReflectionContext();
+          ThreadSafeReflectionContext reflection_context =
+              GetReflectionContext();
```

Comment:
How long does a MemoryReader live?

Reply:
Nevermind, this is inside a lambda and will get immediately released. This is fine!